lib/jekyll/convertible.rb

```diff
@@ -193,7 +193,7 @@ def invalid_layout?(layout)
     # Returns nothing
     def render_all_layouts(layouts, payload, info)
       _renderer.layouts = layouts
-      _renderer.place_in_layouts(output, payload, info)
+      _renderer.place_in_layouts(payload, info)
```

Comment:
We can't change any API signatures during a rubocop refactor. We don't know what plugins use this method and don't want to break them. Please revert this change!